### ISSUE TITLE
1.6 gives me SyntaxError: invalid syntax

### ISSUE THREAD

**Issue by clauseggers** [michael-lazar/rtv#165]
```
Traceback (most recent call last):
  File "/usr/bin/rtv", line 7, in <module>
    from rtv.__main__ import main
  File "/usr/lib/python3.4/site-packages/rtv/__main__.py", line 18, in <module>
    from .oauth import OAuthTool
  File "/usr/lib/python3.4/site-packages/rtv/oauth.py", line 6, in <module>
    from tornado import gen, ioloop, web, httpserver
  File "/usr/lib64/python3.4/site-packages/tornado/gen.py", line 86, in <module>
    from tornado.concurrent import Future, TracebackFuture, is_future, chain_future
  File "/usr/lib64/python3.4/site-packages/tornado/concurrent.py", line 37, in <module>
    from concurrent import futures
  File "/usr/lib/python3.4/site-packages/concurrent/futures/__init__.py", line 8, in <module>
    from concurrent.futures._base import (FIRST_COMPLETED,
  File "/usr/lib/python3.4/site-packages/concurrent/futures/_base.py", line 355
    raise type(self._exception), self._exception, self._traceback
                               ^
SyntaxError: invalid syntax
```

**Comment by shaggytwodope** [comment]
What distro are you using, complete install method and at what point does this error occur for you?

**Comment by clauseggers** [comment]
OpenSUSE Tumbleweed, installed with pip. This is the traceback I get when entering rtv in the prompt.

**Comment by shaggytwodope** [comment]
@clauseggers the links below shed some light on what I think is your issue. For some reason the packages installed just arent right.

I'am wondering if somehow the ``sys.version_info`` check in setup.py is causing futures to be installed for python3 when it shouldnt be with pip? (probally crazy talk, it may be something else entirely)

Some reference for @michael-lazar 

https://github.com/datastax/python-driver/pull/387

https://github.com/agronholm/pythonfutures/issues/41

**Comment by michael-lazar** [comment]
From what I've found, it looks like the python wheel isn't respecting the ``sys.version_info`` check in **setuptools.py**. May need to add an extra conditional dependency via https://wheel.readthedocs.org/en/latest/index.html#defining-conditional-dependencies

@clauseggers The long-term solution is for us to fix the distribution to correctly skip this dependency when installing on python3. The short-term workaround for you is to uninstall the futures dependency in python3. 
```
pip3 uninstall futures
```

**Comment by michael-lazar** [comment]
Ok this is fixed in v1.6.1. You may still need to run ``pip3 uninstall futures`` to remove the dependency from your machine, but it shouldn't be a problem in the future and for other people. Thanks for the help @shaggytwodope and @clauseggers
